### DISCUSSION TITLE
EBICS 3.0 H005 supported ?

### DISCUSSION THREAD

**Discussion by MichaelDesb** [libfintx/libfintx#30] [Q&A]
First, this is a wonderful library. It took me very little time to be able to send INI, HIA and HPB.
But when I sent HPB i checked the answer and the keys I received where from version 2.5. I checked the implementation and it seems the schemas XSd of the version 3.0 is integrated, but not in the commands.

Will it be integrated in the future ? I dont think the 2.5 version will be supported for a long time.

**Comment by leo-schick** [comment]
Yes I think this should be integrated. Would you be interested in forking the repo and creating a Pull Request for it?

I currently maintain FinTS. @torstenprivate  is working on EBICS. He is often quite busy so I can't tell when this is likely to be solved. Contributions are always welcome ;-)

**Comment by jhe74** [comment]
Hi @leo-schick, this discussion was opened 6 months ago. 
Any chance that the EBICS version 3 is included in the future? Or, do you know another library which run the version 3?

Cheers

**Reply by leo-schick** [reply]
Hi @jhe74 ,
I have to say that I don't use EBICS but FinTS. The EBICS implementation has been maintained by @iamtorsten . He seems to be busy since he did not reply yet. I can say that contributions are heartly welcome, so, if you would like to have this functionaliy, feel free, create [a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo), implement it, test it and I will be happy to merge it 👍 It looks like that you can get access to the EBICS standard documentation [here](https://www.ebics.de/de/ebics-standard).
